versionedRouter.js

```diff
@@ -126,6 +128,47 @@ async function handleCdkV2(destName, parsedEvent, flowType) {
   }
 }
 
+async function compareWithCdkV2(destName, event, flowType, v0Result) {
+  try {
+    const cdkOutput = await handleCdkV2(destName, event, flowType);
+    const unmatchedKeys = Object.keys(
+      CommonUtils.objectDiff(v0Result.output, cdkOutput)
+    );
+    if (unmatchedKeys.length > 0) {
+      logger.error(
+        `[LIVE_COMPARE_TEST] destName=${destName}, flowType=${flowType}, unmatchedKeys=${unmatchedKeys}, metadata=${JSON.stringify(
+          event.metadata
+        )}`
+      );
```

Comment:
We will be using loki to monitor this right? What about other logs which we have they might pollute also what about metrics if we want to see data points in grafana?

Reply:
Right now decided to not put any metrics, we can do that later.

Reply:
But how we will measure how many events were not equal ?

Reply:
we need to make sure that there are not error in logs, ideally all events should be equal

Reply:
@sanpj2292 suggested to try out Prometheus client to push metrics but it is not working properly so need to investigate and it is not blocker we can do that post this.